ndbench-dynamodb-plugins/src/main/java/com/netflix/ndbench/plugin/dynamodb/configs/DynamoDBConfigs.java

```diff
@@ -0,0 +1,42 @@
+package com.netflix.ndbench.plugin.dynamodb.configs;
+
+import com.netflix.archaius.api.annotations.Configuration;
+import com.netflix.archaius.api.annotations.DefaultValue;
+
+/**
+ * Configurations for DynamoDB benchmarks
+ *
+ * @author ipapapa
+ */
+@Configuration(prefix = "ndbench.config.dynamodb")
+public interface DynamoDBConfigs {
+
+	@DefaultValue("ndbench-table")
+	String getTableName();
+
+	/*
+	 * Attributes – Each item is composed of one or more attributes. An attribute is
```

Comment:
can you elaborate on what is an item?

Reply:
https://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/com/amazonaws/services/dynamodbv2/document/Item.html